src/ng-select/ng-select.component.ts

```diff
@@ -421,10 +395,29 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, ControlV
         }
     }
 
-    private selectWriteValue(value: any) {
-        let item = this.itemsList.findItem(value, this.bindLabel, this.bindValue);
-        if (item) {
-            this.itemsList.select(item);
+    private selectWriteValue(value: any | any[]) {
+        if (!value) {
+            return;
+        }
+
+        const select = (val: any) => {
+            let item = this.itemsList.findItem(val);
+            if (item) {
+                this.itemsList.select(item);
+            } else {
+                if (!this.bindValue) {
```

Comment:
this won't work for bindValue as we discussed?

Reply:
yes, we will be able to support this with simple arrays